### PR TITLE
Add my module - Piadozo

### PR DESCRIPTION
**Pull request by piadozo-edriane**
It contains:
1. Who am I
2. What's your thought to me as a teammate :>
3.  Rate my function

### CODE REVIEW

File: ourpackage/piadozo.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import random
+
+def edriane():
+    while True:
+        os.system('cls')
+        print ("Your Teammate :)")
+        print ("1. Who am I ?")
+        print ("2. What's your thought to me as a teammate :>")
+        print ("3. Rate my function")
+        print ("0. Exit")
```
Comment (barcelos-kevin) [inline ourpackage/piadozo.py:15]: I will add my comment/changes. - Barcelos

**Review by barcelos-kevin — CHANGES_REQUESTED** [review]

File: ourpackage/piadozo.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+def edriane():
+    while True:
+        os.system('cls')
+        print ("Your Teammate :)")
+        print ("1. Who am I ?")
+        print ("2. What's your thought to me as a teammate :>")
+        print ("3. Rate my function")
+        print ("4. Comment from Barcelos. ")
+        print ("0. Exit")
```
Comment (veslino-marc) [inline ourpackage/piadozo.py:15]: I will add my comments. - Veslino

**Review by veslino-marc — CHANGES_REQUESTED** [review]

File: ourpackage/piadozo.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+def edriane():
+    while True:
+        os.system('cls')
+        print ("Your Teammate :)")
+        print ("1. Who am I ?")
+        print ("2. What's your thought to me as a teammate :>")
+        print ("3. Rate my function")
+        print ("4. Comment from Barcelos. ")
+        print ("5. Comment from Veslino.")
+        print ("0. Exit")
```
Comment (consultado-kirby) [inline ourpackage/piadozo.py:15]: I will add my comment. - Consultado

**Review by consultado-kirby — CHANGES_REQUESTED** [review]

File: ourpackage/piadozo.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    while True:
+        os.system('cls')
+        print ("Your Teammate :)")
+        print ("1. Who am I ?")
+        print ("2. What's your thought to me as a teammate :>")
+        print ("3. Rate my function")
+        print ("4. Comment from Barcelos. ")
+        print ("5. Comment from Veslino.")
+        print ("6. Comment from Consultado.")
+        print ("0. Exit")
```
Comment (andaya-gener) [inline ourpackage/piadozo.py:15]: I will add my comment. - Andaya

**Review by andaya-gener — CHANGES_REQUESTED** [review]

File: ourpackage/piadozo.py (hunk 1 above)
Comment (piadozo-edriane) [reply ourpackage/piadozo.py:15]: You have done it :)

File: ourpackage/piadozo.py (hunk 2 above)
Comment (piadozo-edriane) [reply ourpackage/piadozo.py:15]: You have done it :)

File: ourpackage/piadozo.py (hunk 3 above)
Comment (piadozo-edriane) [reply ourpackage/piadozo.py:15]: You have done it :)

File: ourpackage/piadozo.py (hunk 4 above)
Comment (piadozo-edriane) [reply ourpackage/piadozo.py:15]: You have done it :)

**Review by andaya-gener — APPROVED** [review]

**Review by veslino-marc — APPROVED** [review]

**Review by consultado-kirby — APPROVED** [review]

**Review by barcelos-kevin — APPROVED** [review]

**Review by andaya-gener — APPROVED** [review]

**Pull request merged**